src/types/venues.ts

```diff
@@ -181,6 +182,14 @@ export interface JazzbarVenue extends Venue {
   };
 }
 
+export interface EmbeddableVenue extends Venue {
+  template: VenueTemplate.embeddable;
+  iframeUrl: string;
+  containerStyles: Record<string, string>;
+  iframeStyles: Record<string, string>;
+  iframeOptions: Record<string, string>;
+}
```

Comment:
We can't guarantee that any of these will be defined, so the types should properly account for that.

```suggestion
export interface EmbeddableVenue extends Venue {
  template: VenueTemplate.embeddable;
  iframeUrl?: string;
  containerStyles?: Record<string, string>;
  iframeStyles?: Record<string, string>;
  iframeOptions?: Record<string, string>;
}
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1243/commits/15fbfa1f5ea82fd3369f63187cae4bcd1f10d06f